frontends/ol-components/src/components/RoutedDrawer/RoutedDrawer.tsx

```diff
@@ -1,18 +1,25 @@
 import React, { useCallback, useEffect, useMemo } from "react"
 import Drawer from "@mui/material/Drawer"
+import styled from "@emotion/styled"
 import type { DrawerProps } from "@mui/material/Drawer"
-import ClickAwayListener from "@mui/material/ClickAwayListener"
 import { ActionButton } from "../Button/Button"
-import { RiCloseLine } from "@remixicon/react"
+import { RiCloseLargeLine } from "@remixicon/react"
 import { useSearchParams } from "react-router-dom"
 import { useToggle } from "ol-utilities"
 
 const closeSx: React.CSSProperties = {
   position: "absolute",
-  top: "0px",
-  right: "0px",
+  top: "16px",
+  right: "22px",
 }
 
+const CloseIcon = styled(RiCloseLargeLine)`
+  &&& {
```

Comment:
Does &&& increase the precedence? Do we need it?

Reply:
It does and we do. We have the `IconContainer` emotion classname and `& .MuiSvgIcon-root` in (Button.tsx#L127](https://github.com/mitodl/mit-open/blob/612c4fad7e419102031b3ab3dbc549093f0f5a8b/frontends/ol-components/src/components/Button/Button.tsx#L127)

Produces e.g. `.css-1qy38u1 .MuiSvgIcon-root`.

`&&&` Gives is the precedence e.g. `.css-r92gtm.css-r92gtm.css-r92gtm`. We actually only need 2 here `&&` but a bit safer and unintended change easy to miss. We may want all close icons to be this size though?